sherpa/astro/ui/utils.py

```diff
@@ -11193,6 +11194,67 @@ def get_arf_plot(self, id=None, resp_id=None, recalc=True):
         plotobj.prepare(arf, data)
         return plotobj
 
+
+    def get_rmf_plot(self, id=None, resp_id=None, recalc=True):
+        """Return the data used by plot_arf.
+
```

Comment:
```suggestion

        .. versionadded:: 4.16.0

```

Not sure I've got the indenttion/spacing correct